add python script for task 4 implementation

stratiiv opened the pull request:
# Description
A python script that finds all the files that contain given text and returns filenames 

Fixes issue #4

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)


# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

# How to run
`python 4_Python -p path -s string`

Vitaliy-Kost commented:
hi, maybe we need an instruction how to run your script?
![image](https://github.com/stratiiv/devops-team-green/assets/146471569/d93d1739-7d6e-4557-960c-979ba58541fa)

stratiiv commented:
> hi, maybe we need an instruction how to run your script? ![image](https://user-images.githubusercontent.com/146471569/272174066-d93d1739-7d6e-4557-960c-979ba58541fa.png)

fixed the bug and added instructions
